docs/DeveloperGuide.md

```diff
@@ -374,19 +371,18 @@ Use case ends.
 1. User !!lists contacts(UC2)!!.
 2. User requests to edit a contact.
 3. User specifies the index of the contact and details to edit.
-4. System updates the contact.
-
-Use case ends.
+4. AB3 updates the contact.<br>
+   Use case ends.
 
 **Extensions**
 
-1a. System displays no contacts. <br>
-&emsp;Use case ends. <br>
-3a. System detects user input is invalid. <br>
-&emsp;3a1. System displays and error message. <br>
-&emsp;3a2. User enters new input. <br>
-&emsp;Steps 3a1-3a2 are repeated until user input is valid. <br>
-&emsp;Use case resumes from step 3. <br>
+<div class="step">1a. AB3 displays no contacts.</div>
+<div class="sub-step">Use case ends.</div>
+<div class="step">3a. AB3 detects user input is invalid.</div>
+<div class="sub-step">3a1. AB3 displays an error message.</div>
+<div class="sub-step">3a2. User enters new input.</div>
+<div class="sub-step">Steps 3a1-3a2 are repeated until user input is valid.</div>
+<div class="sub-step">Use case resumes from step 3.</div>
```

Comment:
Should this be `Use case resumes from step 4.` instead?

Reply:
Fixed